homeassistant/components/alexa/handlers.py

```diff
@@ -928,3 +956,92 @@ def temperature_from_object(hass, temp_obj, interval=False):
     )
 
     return response
+
+
+@HANDLERS.register(("Alexa.ChannelController", "ChangeChannel"))
+async def async_api_changechannel(hass, config, directive, context):
+    """Process a change channel request."""
+    channel = "0"
+    entity = directive.entity
+    payload = directive.payload["channel"]
+    payload_name = "number"
+
+    if "number" in payload:
+        channel = payload["number"]
+        payload_name = "number"
+    elif "callSign" in payload:
+        channel = payload["callSign"]
+        payload_name = "callSign"
+    elif "affiliateCallSign" in payload:
+        channel = payload["affiliateCallSign"]
+        payload_name = "affiliateCallSign"
+    elif "uri" in payload:
+        channel = payload["uri"]
+        payload_name = "uri"
+
+    data = {
+        ATTR_ENTITY_ID: entity.entity_id,
+        media_player.const.ATTR_MEDIA_CONTENT_ID: channel,
+        media_player.const.ATTR_MEDIA_CONTENT_TYPE: "channel",
+    }
+
+    await hass.services.async_call(
+        entity.domain,
+        media_player.const.SERVICE_PLAY_MEDIA,
+        data,
+        blocking=False,
+        context=context,
+    )
+
+    response = directive.response()
+
+    response.add_context_property(
+        {
+            "namespace": "Alexa.ChannelController",
+            "name": "channel",
+            "value": {payload_name: channel},
+        }
+    )
+
+    return response
+
+
+@HANDLERS.register(("Alexa.ChannelController", "SkipChannels"))
+async def async_api_skipchannel(hass, config, directive, context):
+    """Process a skipchannel request."""
+    channel = int(directive.payload["channelCount"])
+    entity = directive.entity
+    sleep_delay = 0.5
+
+    data = {ATTR_ENTITY_ID: entity.entity_id}
+
+    for _ in range(0, abs(channel)):
+        if channel > 0:
+            await hass.services.async_call(
+                entity.domain,
+                SERVICE_MEDIA_NEXT_TRACK,
```

Comment:
Define the service name in a variable prior to executing the range instead of defining two code blocks.